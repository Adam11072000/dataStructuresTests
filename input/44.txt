init 67
quit