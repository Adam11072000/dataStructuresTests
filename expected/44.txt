init done.
resetLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
getHighestScoredLabel: FAILURE
addImage: INVALID_INPUT
resetLabelScore: INVALID_INPUT
setLabelScore: FAILURE
mergeSuperPixels: SUCCESS
addImage: SUCCESS
setLabelScore: SUCCESS
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
quit done.
